lib/arena.rb

```diff
@@ -1,2 +1,36 @@
 class Arena
+  attr_accessor :name, :gladiators
+
+  def initialize(name)
+    @name = name
+    @gladiators = []
+  end
+
+  def name
+    @name.capitalize
```

Comment:
I think you could just do @name = name.capitalize instead in the initialize method